CMakeModules/InstallDesktopFiles.cmake

```diff
@@ -0,0 +1,4 @@
+include(GNUInstallDirs)
+
+install (FILES ${CMAKE_SOURCE_DIR}/resources/desktop/plexmediaplayer.desktop DESTINATION ${CMAKE_INSTALL_FULL_DATAROOTDIR}/applications)
```

Comment:
I believe that this file sould be InstallLinuxDesktopFile which helps knowing it's about linux at first look

Reply:
Yep, that sounds like a good idea